src/VisualStudio/LiveShare/Impl/Client/RemoteLanguageServiceWorkspaceAccessor.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.LanguageServices.ExternalAccess.VSTypeScript.Api;
+
+namespace Microsoft.VisualStudio.LanguageServices.LiveShare.Client
```

Comment:
❗ This should be in the VSTypeScript folder and namespace

Reply:
Moved - let me know if I over-complicated it :)